rust/kcl-lib/tests/execute_engine_error_return/execution_error.snap

```diff
@@ -2,24 +2,32 @@
 source: kcl-lib/src/simulation_tests.rs
 description: Error from executing execute_engine_error_return.kcl
 ---
-KCL Engine error
+KCL Semantic error
 
-  × engine: The path is not closed.  Solid2D construction requires a closed
```

Comment:
No longer an engine error. Now KCL catches it and produces a semantic error before the engine is even called.